packages/std/src/math/decimal.rs

```diff
@@ -20,6 +21,10 @@ impl Decimal {
     const DECIMAL_FRACTIONAL: Uint128 = Uint128::new(1_000_000_000_000_000_000u128); // 1*10**18
     const DECIMAL_FRACTIONAL_SQUARED: Uint128 =
         Uint128::new(1_000_000_000_000_000_000_000_000_000_000_000_000u128); // (1*10**18)**2 = 1*10**36
+    const DECIMAL_FRACTIONAL_UINT256: Uint256 = Uint256::from_be_bytes([
```

Comment:
This bugs me a bit, but then again I don't have a better idea.

Reply:
We can have a const `Uint128 -> Uint256` converter. This just needs its own API as the Fram implementation cannot be done as const.

Reply:
Tracked in https://github.com/CosmWasm/cosmwasm/issues/1107. This is out of scope for this PR and can easily be added later on.